lib/src/chart/bar_chart/bar_chart_data.dart

```diff
@@ -488,9 +488,15 @@ class BarChartRodStackItem with EquatableMixin {
   BarChartRodStackItem(
     this.fromY,
     this.toY,
-    this.color, [
+    this.color,
+    this.label,
+    this.labelStyle, // New property
```

Comment:
Ahh, I see it is not optional. Maybe we can make them optional (like nullable named parameter, or optional positional argument)
Also please remove the comment

Reply:
Also, it seems you forgot to add these properties in `List<Object?> get props`